rero_ils/static/scss/rero_ils/header.scss

```diff
@@ -6,32 +6,62 @@
  * under the terms of the MIT License; see LICENSE file for more details.
  */
 
-
 .rero-ils-logo {
-    max-width: 5em;
-    margin: 7px 0;
+  max-height: $navbar-brand-height;
+  margin: 7px 0;
+}
+
+header {
+  position: relative;
 }
 
-.navbar {
-    min-height: 4em;
+// .navbar {
+//     min-height: 4em;
+// }
+
+.rero-ils-page-body {
+  height: calc(100vh - 0px);
+  overflow: auto;
+}
+
+.nav-link, .navbar-toggler:focus {
+  outline: 0;
+}
+
+.nav-item a.collapsed .fa-caret-down {
+  transform: rotate(-90deg);
+}
+
+.list-group-item:last-child {     //TODO: understand why it's needed, it shouldn't
+  border-bottom: 0;
+}
+
+@media (min-width: 768px) {
+  #mobileHide {
+    position: fixed;
+    top: 0;
+    z-index: 1000;
+    width: 60%;
+    border: solid lightgray 1px;
+  }
 }
 
 
 // Had to implement these colors, but couldn't understand why.
 // These rules set the sidebar links colors too.
 
-.navbar-inverse a {
-  color: $navbar-inverse-link-color;
+/*.navbar-inverse a {
```

Comment:
Where the comment is closed?

Reply:
At the end of the file. I forgot to remove the comment.